protos/feast/core/JobService.proto

```diff
@@ -68,42 +68,26 @@ enum JobStatus {
 message Job {
   // Identifier of the Job
   string id = 1;
-  // External Identifier of the Job assigned by the Spark executor
-  string external_id = 2;
   // Type of the Job
-  JobType type = 3;
+  JobType type = 2;
   // Current job status
-  JobStatus status = 4;
-  // Timestamp on when the job was is created
-  google.protobuf.Timestamp created_timestamp = 5;
-  // Timestamp on when the job has stopped.
-  google.protobuf.Timestamp stop_timestamp = 6;
-
-  message ExportJobMeta {
-    // Glob of the exported files that should be retrieved to reconstruct
-    // the dataframe with retrieved features.
-    repeated string file_glob = 1;
-    // The Historical Features request that triggered this export job
-    GetHistoricalFeaturesRequest request = 2;
```

Comment:
Removed a bunch of unused (or not yet used) parameters here, we can always add them back later.